extensions/amp-analytics/0.1/vendors.js

```diff
@@ -438,6 +438,35 @@ export const ANALYTICS_CONFIG = {
         '${actionParameter}${parameterSuffix}',
     },
   },
+
+  'shinystat': {
+    'requests': {
+      'ssurl': 'https://amp.shinystat.com/cgi-bin/getcod.cgi',
+      'ssparam': 'AMP=1' +
+                 '&USER=${account}' +
+                 '&AMPPAG=${page}' +
+                 '&AMPHR=${canonicalUrl}' +
+                 '&AMPREFER=${documentReferrer}' +
+                 '&AMPRES=${screenWidth}X${screenHeight}' +
+                 '&AMPCOLOR=${screenColorDepth}' +
+                 '&AMPCID=${clientId(AMP_CID)}' +
+                 '&AMPPID=${pageViewId}' +
+                 '&AMPTTL=${title}' +
+                 '&AMPRQC=${requestCount}' +
+                 '&AMPVIE=${viewer}' +
+                 '&AMPSSIDC=${queryParam(ssidc)}' +
+                 '&${ampp1}=${ampv1}' +
+                 '&${ampp2}=${ampv2}',
+      'pageview': '${ssurl}?${ssparam}',
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true,
```

Comment:
My company signed the CLA. I quote my one week old answer:

> Thanks for your help. We tested it successfully. Safari allows the javascript code to access session and local storage data. Even though it deletes everything on exit, data persist long enough to collect the informations we need.

Is there any other issue?